lib/foreman_ansible/engine.rb

```diff
@@ -93,6 +93,7 @@ class Engine < ::Rails::Engine
       ::Api::V2::HostgroupsController.include ForemanAnsible::Api::V2::HostgroupsControllerExtensions
       ::Api::V2::HostgroupsController.include ForemanAnsible::Api::V2::HostgroupsParamGroupExtensions
       ::ConfigReportImporter.include ForemanAnsible::AnsibleReportImporter
+      ::ForemanRemoteExecution::Renderer::Scope::Input.prepend ForemanAnsible::Renderer::Scope::AnsibleTagsMacro
```

Comment:
Why are we not using the plugin [extension point](https://github.com/theforeman/foreman/blob/e49da5764920b3810e1c33ea2517ec8c870362b8/app/registries/foreman/plugin.rb#L434) for this?

Reply:
Because we extend the scope from rex here rather than the one from core.

Reply:
Right, I'd rather see us having similar way of extension, but you are right, it is not there now.
just we are patching module and that might fail when include of that module happens prior to the patch, so this is bit fragile.

Reply:
Any suggestions for a better approach? By the way, which module do you mean? `::ForemanRemoteExecution::Renderer::Scope::Input` is a [class](https://github.com/theforeman/foreman_remote_execution/blob/master/app/lib/foreman_remote_execution/renderer/scope/input.rb#L4)...